content/docs/user-guide/packaging/project-export-cli.md

```diff
@@ -0,0 +1,254 @@
+---
+linkTitle: Project Export CLI
+title: Project Export CLI Tool
+description: Learn how to use the Project Export CLI to automate preparing your project for release.
+toc: true
+weight: 400
+---
+With the new `export-project` CLI command, it is now possible to automate the packaging and release of game projects.
```

Comment:
‘’’suggestion
The `export-project` CLI command is used to package game code and assets for a release.
‘’’